### PR TITLE
Set the correct temporary path for minijail chroot.

### PR DESCRIPTION
**Pull request by oliverchang**

### CODE REVIEW

**Comment by oliverchang** [comment]
TBR. /gcbrun

File: src/python/bot/fuzzers/libFuzzer/engine.py [hunk 1]
```diff
@@ -53,6 +53,11 @@ def __init__(self, corpus_dir, arguments, strategies, fuzz_corpus_dirs,
     self.is_mutations_run = is_mutations_run
 
 
+def _get_runner(target_path):
+  """Get the libFuzzer runner."""
+  return libfuzzer.get_runner(target_path, temp_dir=fuzzer_utils.get_temp_dir())
```
Comment (inferno-chromium) [inline src/python/bot/fuzzers/libFuzzer/engine.py]: Why not do inside libfuzzer.get_runner, that if temp_dir is None, set it to fuzzer_utils.get_temp_dir()

Comment (oliverchang) [reply src/python/bot/fuzzers/libFuzzer/engine.py]: sure, done.

**Pull request merged**